src/applications/login/containers/SignInApp.jsx

```diff
@@ -59,7 +55,9 @@ class SignInPage extends React.Component {
 
   render() {
     const { globalDowntime } = this.state;
-    const loggedOut = this.props.location.query.auth === 'logged_out';
+    const { query } = this.props.location;
+    const loggedOut = query.auth === 'logged_out';
+    const application = query.application;
```

Comment:
this gives us flexibility to direct to other applications or if the `myvahealth` param were to change, too